packages/zoe/src/contracts/multipoolAutoswap.js

```diff
@@ -283,7 +285,10 @@ export const makeContract = harden(zcf => {
     let tempLiqHandle;
 
     const tempLiqOfferHook = tmpHandle => (tempLiqHandle = tmpHandle);
-    const tempLiqInvite = zcf.makeInvitation(tempLiqOfferHook);
+    const tempLiqInvite = inviteAnOffer({
```

Comment:
Hmm, there's no reason to use inviteAnOffer here, since we don't have `expected`.